src/components/common/messaging/MessageBox.tsx

```diff
@@ -45,6 +45,8 @@ import { PermissionTooltip } from "../Tooltip";
 import FilePreview from "./bars/FilePreview";
 import ReplyBar from "./bars/ReplyBar";
 
+import { DraftObject }from "../../../mobx/stores/Draft";
```

Comment:
Minor linting nitpick: 
```suggestion
import { DraftObject } from "../../../mobx/stores/Draft";
```